analyzers/src/SonarAnalyzer.CSharp/Rules/InvalidCastToInterface.cs

```diff
@@ -35,6 +35,7 @@ public sealed class InvalidCastToInterface : SonarDiagnosticAnalyzer
         private const string MessageReviewFormat = "Review this cast; in this project there's no type that {0}.";
 
         public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics { get; } = ImmutableArray.Create(InvalidCastToInterfaceRuleConstants.Rule);
+        protected override bool ParallelProcessingDisabled => true;
```

Comment:
I would call the Property Concurrent instead of Paralell. That is the name in the Roslyn API too.